src/inc/quic_crypt.h

```diff
@@ -40,6 +40,11 @@ typedef struct QUIC_HASH QUIC_HASH;
 //
 #define QUIC_IV_LENGTH 12
 
+//
+// The maximum buffer length of the IV need by the platform layer.
+//
+#define QUIC_MAX_IV_LENGTH 48
```

Comment:
Should we special case this to the Windows platform, since BCrypt is the only one that has this behavior(bug)? Everywhere else, `QUIC_IV_LENGTH` suffices.

Reply:
I would prefer that since this is a platform specific workaround.